assets/src/components/detours/diversionPage.tsx

```diff
@@ -171,7 +182,11 @@ export const DiversionPage = ({
     }
   }
 
-  const detourPanel = () => {
+  const detourPanel: ({
+    deleteDetourCallback,
+  }: {
+    deleteDetourCallback: () => void
+  }) => React.JSX.Element = () => {
```

Comment:
question: What's the benefit of getting this via a parameter instead of pulling it from the scope?